submissions/chernetskyi8704/Hooli-style Popup/index.html

```diff
@@ -0,0 +1,181 @@
+                <ul class="popup__list-more">
+                  <li class="popup__item">
+                    <a class="popup__link link" href=""
+                      ><img class="popup__img" src="img/calendar.png" alt="" />
+                      <p class="link__text">Calendar</p></a
+                    >
+                  </li>
+                  <li class="popup__item">
+                    <a class="popup__link link" href=""
+                      ><img class="popup__img" src="img/mail.png" alt="" />
+                      <p class="link__text">Mail</p></a
+                    >
+                  </li>
+                  <li class="popup__item">
+                    <a class="popup__link link" href=""
+                      ><img class="popup__img" src="img/news.png" alt="" />
+                      <p class="link__text">News</p></a
+                    >
+                  </li>
+                  <li class="popup__item">
+                    <a class="popup__link link" href=""
+                      ><img class="popup__img" src="img/maps.png" alt="" />
+                      <p class="link__text">Map</p></a
+                    >
+                  </li>
+                  <li class="popup__item">
+                    <a class="popup__link link" href=""
+                      ><img class="popup__img" src="img/files.png" alt="" />
+                      <p class="link__text">Files</p></a
+                    >
+                  </li>
+                  <li class="popup__item">
+                    <a class="popup__link link" href=""
+                      ><img class="popup__img" src="img/contacts.png" alt="" />
+                      <p class="link__text">Contacts</p></a
+                    >
+                  </li>
+                  <li class="popup__item">
+                    <a class="popup__link link" href=""
+                      ><img class="popup__img" src="img/classroom.png" alt="" />
+                      <p class="link__text">Classroom</p></a
+                    >
+                  </li>
+                  <li class="popup__item">
+                    <a class="popup__link link" href=""
+                      ><img
+                        class="popup__img"
+                        src="img/tanslation.png"
+                        alt=""
+                      />
+                      <p class="link__text">Translation</p></a
+                    >
+                  </li>
+                  <li class="popup__item">
+                    <a class="popup__link link" href=""
+                      ><img class="popup__img" src="img/documents.png" alt="" />
+                      <p class="link__text">Documents</p></a
+                    >
+                  </li>
+                </ul>
+              </div>
+            </div>
+          </li>
+          <li class="nav__item">
+            <a href="" class="nav__link link"
+              ><img class="nav__img" src="img/bell.png" alt="Bell"
```

Comment:
What about those icons?

Reply:
Thanks for another comment. This Bell image is decorative too so I deleted the alt attribute, but the next image inside the link that without text content, so I decided to leave the alt attribute, am I correct?

Reply:
if you want to keep this description, it is more appropriate to transfer it to the `aria-label` attribute of the link.
Try to stick to one approach when using/creating elements).
So better to populate it with an empty value.

Reply:
Thank you very much for the advice, I will take it into account in the future. Corrected :)

Reply:
I tried to make 2 committees into 1, and 4 came out, but then I fixed everything and this is what happened, although there are no conflicts locally. What does that mean?
![image](https://user-images.githubusercontent.com/78321800/185696359-e1799aeb-c3aa-45b0-9ca6-8dbeee059d2d.png)